about.html

```diff
@@ -0,0 +1,121 @@
+                <li class="second-navbar-links"><a href="#"></a> login</li>
+            </ul>
+        </div>
+        </nav>
+        <div class="banner-container">
+        <div class="heading">
+            <h4 class="heading-1">Hello!!!</h4>
+            <h1 class="heading-2"> Welcome To Online Makeup Blog </h1>
+        </div>
+        <div>
+            <p  class="paragraph">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Explicabo, libero?
+            </p>
+            <div class="contact-container">
+            <span class="contact">please contact us via email if you have any further questions abot my capstone PR.</span> <span class="gmail">Alizaamir49@gmail.com</span>
+        </div>
+        </div>
+    
+        </div>
+    </header>
+<div class="after-about flex" >
+
+    <div class="after-about-heading flex">
+        <h2 class="heading1 center1"> Creative Makeup Logo</h2>
+        
+    <span class="red-bar flex">
+    </span>
+</div>
+<div class="about-para">
+    <p class="about-paragraph">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Dignissimos natus, laboriosam dolorem nostrum possimus provident doloribus aspernatur ullam excepturi earum.
+    </p>
+</div>
+<div class="about-img">
+    <img src="./images/logo.jpg" alt="" class="logo-about">
+</div>
+</div>
+     <div class="after-about-para">
+        <div class="makeup-histroy flex">
+            <h2 class="heading2">See Makeup Hisytroy</h2>
+            <Span class="red-bar flex"></Span>
+        </div>
+        <div class="makeup-histroy-para">
+            <p class="center"> Lorem ipsum dolor sit, amet consectetur adipisicing elit. Explicabo quisquam dolores, distinctio ea in beatae.</p>
+        </div>
+        <div class="histroy-card">
+<div class="card1">
+    <img src="./images/card1.png" alt="" class="card-img">
+</div>
+        </div>
+        <div class="card1">
+            <img src="./images/card2.png" alt="" class="card-img">
+        </div>
+       
+        </div>
+<section class="partners">
+    <h2 class="partner-heading flex">
+        Partners
+        <span class="red-bar flex">
+
+        </span>
+
+    </h2>
+    <div class="partner-logo flex">
+        <img src="./images/mozilla 1.png" alt="">
+        <img src="./images/googlekorea 1.png" alt="">
+        <img src="./images/mozilla 1.png" alt="">
+        <img src="./images/naver 1.png" alt="">
+        <img src="./images/googlekorea 1.png" alt="">
+      </div>
+</section>
+     <footer class="footer flex">
+        <div class="footer1 flex">
+            <div class="footer-logo">
+                <img src="./images/logo.jpg" alt="" class="footer-img">
+            </div>
+        <div class="footer-para">
+            <p class="footer-para-content">2023 Capstone Project <br> No Rights Reserved</p>
+        </div>
+        </div>
+
+     </footer>
```

Comment:
- [ ] Based on the project requirement, the footer section of the **About** page when viewed on the desktop, the footer should have a dark background and the content should be centered. Kindly fix this

![image](https://github.com/alizaamir123/Capstone-project/assets/84156714/580edaaa-304a-4e87-91b6-0376c1a9cdef)